CommonMark/Parser/BlockMethods.cs

```diff
@@ -1,954 +1,985 @@
 ﻿using System.Collections.Generic;
```

Comment:
The indentation has been changed thus most of the file shows up as a change - can you please format the doc with the default indent so that the actual changes get highlighted?